compiler/rustc_middle/src/mir/interpret/error.rs

```diff
@@ -427,6 +427,10 @@ pub enum UndefinedBehaviorInfo<'tcx> {
     AbiMismatchArgument { caller_ty: Ty<'tcx>, callee_ty: Ty<'tcx> },
     /// ABI-incompatible return types.
     AbiMismatchReturn { caller_ty: Ty<'tcx>, callee_ty: Ty<'tcx> },
+    /// Called `const_make_global` on a non-heap pointer.
+    ConstMakeGlobalPtrIsNonHeap,
+    /// Called `const_make_global` twice.
+    ConstMakeGlobalPtrAlreadyImm(AllocId),
```

Comment:
Usually const-eval-only errors are not added here, they are added in the const-eval-specific error type. Is that not possible here for some reason?